src/main/java/com/watch/shop/app/view/WatchView.java

```diff
@@ -0,0 +1,147 @@
+import static com.watch.shop.app.view.ViewConstants.FORMAT_HEADER;
+import static com.watch.shop.app.view.ViewConstants.FORMAT_ROW;
+import static com.watch.shop.app.view.ViewConstants.FORMAT_SEPARATOR;
+import static com.watch.shop.app.view.ViewConstants.MENU;
+import static com.watch.shop.app.view.ViewConstants.SORT_MENU;
+import static com.watch.shop.app.view.ViewConstants.START_SEPARATOR;
+
+public class WatchView {
+
+    private final InputHandler inputHandler = new InputHandler();
+
+    public String getUserChoice() {
+        printMessage(MENU);
+
+        return inputHandler.readLineFromConsole();
+    }
+
+    public String sortByParam() {
+        printMessage(SORT_MENU);
+
+        return inputHandler.readLineFromConsole();
+    }
+
+    public void displayAllWatches(List<Watch> watches) {
+        printMessage(START_SEPARATOR);
+        printFormattedMessage(FORMAT_HEADER, "Brand", "Model", "Color", "Type", "Price", "Arrival Date");
+        printFormattedMessage(FORMAT_SEPARATOR, "_".repeat(12), "_".repeat(26), "_".repeat(11), "_".repeat(11), "_".repeat(11), "_".repeat(13));
+        watches.forEach(w -> printFormattedMessage(FORMAT_ROW, w.getBrand(), w.getModel(), w.getColor(), w.getType(), w.getPrice(), w.getArrivalDate()));
+        printMessage(END_SEPARATOR);
+    }
+
+    public void displayTotalCost(BigDecimal totalCost) {
+        String message = """
+                \nTotal cost:
+                 %.2f""".formatted(totalCost);
+        printMessage(message);
+    }
+
+    public Watch addNewWatch() {
+
+        printMessage("Enter Brand:");
+        String brand = inputHandler.readLineFromConsole();
+
+        printMessage("Enter Model:");
+        String model = inputHandler.readLineFromConsole();
+
+        printMessage("Enter Color (WHITE, SILVER, GOLD, BLUE, RED, BLACK):");
+        String color = inputHandler.readLineFromConsole().toUpperCase();
+
+        printMessage("Enter Type (QUARTZ, MECHANICAL, SOLAR, SMART):");
+        String type = inputHandler.readLineFromConsole().toUpperCase();
+
+        printMessage("Enter Price: ");
+        String price = inputHandler.readLineFromConsole();
+
+        printMessage("Enter Arrival Date (YYYY-MM-DD):");
+        String arrivalDate = inputHandler.readLineFromConsole();
+
+        return createWatchBuilderByType(WatchType.valueOf(type))
+                .brand(brand)
+                .model(model)
+                .color(Color.valueOf(color))
+                .type(WatchType.valueOf(type))
+                .price(new BigDecimal(price))
+                .arrivalDate(LocalDate.parse(arrivalDate))
+                .build();
+    }
+
+    public void printFormattedMessage(String message, Object... args) {
+        System.out.printf(message, args);
+    }
+
+    public void printMessage(String message) {
+        System.out.println(message);
+    }
+
+    private WatchBuilder<? extends Watch, ?> createWatchBuilderByType(WatchType type) {
```

Comment:
all in this class beside System.out.println methods, are controller logic.

Menu, Builders, Watch, call view method, control flow and so on.

Just extract all these methods to Controller package - or private message in controller, or additional class MenuHandler for example.

View on MVC architecture it's component that knows about two things:
input from User
output to User

Entity, Builders, Menu and so on, it's Controller, because it's logic regarding Main_menu